roles/openshift_logging_kibana/templates/kibana.j2

```diff
@@ -143,29 +143,29 @@ spec:
                resourceFieldRef:
                  containerName: kibana-proxy
                  resource: limits.memory
-{% if (openshift_http_proxy is defined and openshift_http_proxy is not none and openshift_http_proxy != "") %}
+{% if 'http_proxy' in openshift.common %}
             -
              name: "http_proxy"
-             value: "{{ openshift_http_proxy }}"
+             value: "{{ openshift.common.http_proxy | default('') }}"
```

Comment:
This means `openshift.common.http_proxy` exists, but the value could somehow be `None` or empty?

Reply:
i believe `None` should cause the default of `''` to be used... correct @sdodson ?

There shouldn't be any harm in providing the empty `http_proxy` variable, it should resolve on the pod the same as if `http_proxy` variable wasn't set.

Reply:
Yes, `None` would cause default of `''`